alignak/http/daemon.py

```diff
@@ -46,6 +46,51 @@
 logger = logging.getLogger(__name__)  # pylint: disable=C0103
 
 
+try:
+    class Pyopenssl(pyOpenSSLAdapter):
+        """
+        Use own ssl adapter to modify ciphers. This will disable vulnerabilities ;)
+        """
+
+        def __init__(self, certificate, private_key, certificate_chain=None, dhparam=None):
+            """
+            Add init because need get the dhparam
+
+            :param certificate:
+            :param private_key:
+            :param certificate_chain:
+            :param dhparam:
+            """
+            super(Pyopenssl, self).__init__(certificate, private_key, certificate_chain)
+            self.dhparam = dhparam
+
+        def get_context(self):
+            """Return an SSL.Context from self attributes."""
+            cont = SSL.Context(SSL.SSLv23_METHOD)
+
+            # override:
+            ciphers = (
+                'ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+HIGH:'
+                'DH+HIGH:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+HIGH:RSA+3DES:!aNULL:'
+                '!eNULL:!MD5:!DSS:!RC4:!SSLv2'
```

Comment:
What is the differnce with 'DEFAULT:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL' ?
Here you are manually listing what you want and then still remove what you don't want.